Dockerfiles/agent/datadog-ecs.yaml

```diff
@@ -8,9 +8,7 @@ config_providers:
   - name: ecs
     polling: true
 
-# Enable APM by setting the DD_APM_ENABLED envvar to true, or override this configuration
 apm_config:
-  enabled: false
```

Comment:
are we sure APM works OOO in ecs?

Reply:
You're right, Fargate should be off. I got over zealous. Removed it.